### ISSUE TITLE
Výchozí styl pro ověření funkčnosti linkování

### ISSUE THREAD

**Issue by jnv** [czechitas/tvorim-web-1#6]
Všiml jsem si, že u propojení stylů s HTML účastnice očekávají, že se něco stane a nemají zpětnou vazbu že styl připojily správně. Napadlo mě, že bychom mohli do souboru se styly přidat nějaký výchozí, po kterém bude vidět, že se něco stalo a v dalším kroku jej přepíšou (např. `font-weight: bold;`  pro celou stránku). Zároveň na to můžeme navázat výkladem o anatomii CSS.

**Comment by iBobik** [comment]
Dobrý nápad.

Ale spíše bych doporučil udělat nějakou lépe viditelnou změnu, bold hodně lidí přehlédne.

**Comment by Kout** [comment]
Ideální je `color: red;` krátké a úderné.

**Comment by iBobik** [comment]
Ano. Ale jen třeba pro nadpisy, přepravit vše nemusí postřehnout. (Beru ohled na některé velmi nekvalitní displeje se kterými tam chodí.)
